content/docs/reference/spec/migration/buildpack-api-0.5-0.6.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Buildpack API 0.5 -> 0.6"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.6) for buildpack API 0.6 for the full list of changes and further details.
+
+### Opt-in layer caching
+
+Buildpacks must now explicitly opt-in to layer re-use by writing `launch`, `build`, or `cache` keys to a new `[types]` table in `<layers>/<layer>.toml` (note that these keys were removed from the top level). If buildpacks do not modify `<layers>/<layer>.toml`, the layer will not be re-used, even if the buildpack in the previous build set any of these keys to `true`.
+
+A Bash buildpack could write something like the following to `<layers>/<layer>.toml` in order to cache a layer:
+
+```bash
+cat >> layer.toml <<EOL
+[types]
+cache = true
```

Comment:
```suggestion
  cache = true
```
I think that the 2 spaces are necessary but I'm not 100% sure.
Anyway, I think it's clearer to have these spaces.

Reply:
After discussion we think the spaces are not necessary.